WordPress/Classes/Models/WPAccount.m

```diff
@@ -128,4 +128,21 @@ - (BOOL)isWPComAccount
     return self.restApi != nil;
 }
 
+#pragma mark - Sorting support
+
+- (NSComparisonResult)compare:(WPAccount *)account
+{
+    BOOL account1IsWPComAccount = [self isWPComAccount];
+    BOOL account2IsWPComAccount = [account isWPComAccount];
```

Comment:
> WordPress.com accounts go first, the rest is considered all the same.

There are only wp.com `WPAccount` objects, so this would always return `NSOrderedSame` unless for some reason the auth token for an account is missing.

The sorting we want is:
1. default account
2. everything else